microsoft/react-native-macos
# Enabling AccessibilityInfo.setAccessibilityFocus for MacOS

#### Please select one of the following

## Summary

When the MacOS fork was made all the AccessibilityInfo functionality was stubbed out because the iOS Accessibility APIs are different from the MacOS APIs. I've filled back out the functionality for setAccessibilityFocus.

## Changelog

[MacOS] [Added] - Implemented AccessibilityInfo.setAccessibilityFocus

## Test Plan

1. Run RNTester
2. Go to the accessibility page
3. Scroll to Set Accessibility Focus (the last entry)
4. Toggle VoiceOver in settings.
5. Move VoiceOver Focus to "Set Accessibility Focus"
6. Hit [Space]
- You should see the focus being moved to "Move focus here on button press".

![Screen Shot 2020-06-12 at 10 25 40](https://user-images.githubusercontent.com/65255457/84529951-2fc99280-ac97-11ea-9566-9babd0d1c3e3.png)
![Screen Shot 2020-06-12 at 10 26 12](https://user-images.githubusercontent.com/65255457/84529959-322bec80-ac97-11ea-8bab-2a599446c1cc.png)


###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-macos/pull/450)
